health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -0,0 +1,701 @@
+        onChangePage={handlePageChange}
+        onChangeRowsPerPage={handleRowsPerPageChange}
+        paginationRowsPerPageOptions={[5, 10, 15, 20]}
+      />
+      {showPopUp && (
+        <PopUp
+          className={"dataMapping"}
+          type={"default"}
+          heading={t("FACILITY_MAPPING_POP_HEADER")}
+          equalWidthButtons={true}
+          children={[
+            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)